uReplicator-Manager/src/main/java/com/uber/stream/kafka/mirrormaker/manager/core/ControllerHelixManager.java

```diff
@@ -1142,16 +1143,17 @@ public synchronized void addTopicToMirrorMaker(
     try {
       LOGGER.info("Trying to add topic: {} to pipeline: {}", topicName, pipeline);
 
+      updateCurrentStatus();
```

Comment:
it's not recommended to do updateCurrentStatus in addTopicToMirrorMaker because this is a blocking operation which might take a long time to finish.